examples/experimental/sling_decorator/sling_decorator/__init__.py

```diff
@@ -0,0 +1,46 @@
+from dagster import Definitions, file_relative_path
+from dagster_embedded_elt.sling import (
+    DagsterSlingTranslator,
+    sling_assets,
+)
+from dagster_embedded_elt.sling.resources import (
+    SlingConnectionResource,
+    SlingResource,
+)
+
+replication_config = file_relative_path(__file__, "../sling_replication.yaml")
+
+sling_resource = SlingResource(
+    connections=[
+        SlingConnectionResource(
+            name="MY_POSTGRES",
+            type="postgres",
+            connection_string="postgres://postgres:postgres@localhost:54321/finance?sslmode=disable",
+        ),
+        SlingConnectionResource(
+            name="MY_DUCKDB",
+            type="duckdb",
+            connection_string="duckdb:///var/tmp/duckdb.db",
+        ),
+    ]
+)
+
+
+@sling_assets(replication_config=replication_config)
+def my_assets(context, sling: SlingResource):
+    yield from sling.replicate(
+        replication_config=replication_config,
```

Comment:
My initial thought when seeing this was wondering why we need to specify `replication_config` in both `sling.replicate` and `@sling_assets`. 

Gonna take a look at the underlying code which will likely explain why, but still wanted to share it.